modules/apps/commerce/commerce-frontend-js/package.json

```diff
@@ -22,17 +23,17 @@
 		"@clayui/tooltip": "3.4.0",
 		"classnames": "2.2.6",
 		"frontend-js-react-web": "*",
-		"moment": "^2.22.2",
-		"prop-types": "15.7.2",
-		"react": "16.12.0",
-		"react-dom": "16.12.0"
+		"frontend-js-web": "*",
+		"prop-types": "15.7.2"
 	},
 	"devDependencies": {
-		"clay-css": "^2.16.2"
+		"clay-css": "^2.16.2",
+		"faker": "^5.1.0"
```

Comment:
Oh, and I stumbled across [a change were we actually removed `faker` a while back because it was used in only one place](https://issues.liferay.com/browse/LPS-108049).